app/api/events.py

```diff
@@ -20,8 +20,53 @@ class Meta:
 
     id = fields.Str(dump_only=True)
     identifier = fields.Str(dump_only=True)
-    name = fields.Str()
-    event_url = fields.Str()
+    name = fields.Str(required=True)
+    email = fields.Str(required=True)
+    event_url = fields.Str(dump_only=True)
+    starts_at = fields.DateTime(required=True)
+    ends_at = fields.DateTime(required=True)
+    timezone = fields.Str()
+    latitude = fields.Float()
+    longitude = fields.Float()
+    logo = fields.Str()
+    location_name = fields.Str()
+    searchable_location_name = fields.Str()
+    description = fields.Str()
+    background_url = fields.Str()
```

Comment:
@LuD1161 all the background image urls should be in a nested field
```
    "background_image": {
             "thumbnail_image_url": "url for that",
             "large_image_url": "url for that",
             "original_image_url": "url for that",
             "icon_image_url": "url for that"
    }
```

Here,
`background_url` => `original_image_url`
`large` => `large_image_url`
`icon` => `icon_image_url`
`thumbnail` => `thumbnail_image_url`

Please rename the database fields also accordingly

See old issue https://github.com/fossasia/open-event-orga-server/issues/3518